doc/api/requester_api.md

```diff
@@ -31,6 +31,29 @@ returns early with value not equal to `LIBSPDM_STATUS_SUCCESS` then the SPDM con
 before attempting establish a new connection.
 <br/><br/>
 
+---
+### libspdm_get_supported_algorithms
+---
+
+### Description
+Sends the `GET_VERSION` and `GET_CAPABILITIES` requests, where GET_CAPABILITIES.Param1[0] is set.
```

Comment:
Is it planned that the next version of SPDM will also support `extended capabilities` in `GET_VERSION`?

Reply:
That is a question for https://github.com/DMTF/SPDM-WG.